plugins/carbon_black_defense/plugin.spec.yaml

```diff
@@ -138,294 +138,923 @@ types:
+      required: false
+    crossproc_count:
+      title: Crossproc Count
+      type: integer
+      description: Count of crossproc events reported by the sensor since last initialization
+      required: false
+    crossproc_effective_reputation:
+      title: Crossproc Effective Reputation
+      type: string
+      description: Effective reputation of the binary on one side of the cross-process action
+      required: false
+    crossproc_effective_reputation_source:
+      title: Crossproc Effective Reputation Source
+      type: string
+      description: Source of the effective reputation for the cross-process
+      required: false
+    crossproc_guid:
+      title: Crossproc GUID
+      type: string
+      description: Unique process identifier of one of the cross-process members
+      required: false
+    crossproc_hash:
+      title: Crossproc Hash
+      type: "[]string"
+      description: MD5 and/or SHA-256 hash(es) of the binaries whose processes are running on one side of the cross-process action
+      required: false
+    crossproc_md5:
+      title: Crossproc MD5
+      type: string
+      description: MD5 hash of the binary on one side of the cross-process action
+      required: false
+    crossproc_name:
+      title: Crossproc Name
+      type: string
+      description: Filesystem path of the binary of one side of the cross-process action
+      required: false
+    crossproc_pid:
+      title: Crossproc PID
+      type: integer
+      description: Process identifier assigned by the operating system to one of the cross-process members
+      required: false
+    crossproc_process_guid:
+      title: Crossproc Process GUID
+      type: string
+      description: Unique identifer of the process on one side of the cross-process action
+      required: false
+    crossproc_sha256:
+      title: Crossproc SHA-256
+      type: string
+      description: SHA-256 hash of the binary on one side of the cross-process action
+      required: false
+    crossproc_target:
+      title: Crossproc Target
+      type: boolean
+      description: True if the process was the target of the cross-process event; false if the process was the actor
+      required: false
+    device_group:
+      title: Device Group
+      type: string
+      description: Sensor group to which the endpoint was assigned when the sensor recorded the event data
+      required: false
+    device_installed_by:
+      title: Device Installed By
+      type: string
+      description: The Carbon Black Cloud user who was logged in to the endpoint when the sensor was installed
+      required: false
+    event_attack_stage:
+      title: Event Attack Stage
+      type: string
+      description: Stage(s) of the cyber kill chain when an attack was terminated by sensor
+      required: false
+    event_timestamp:
+      title: Event Timestamp
+      type: date
+      description: Timestamp reported by the sensor when the event occured
```

Comment:
This will be in ISO 8601. There are other timestamps as well (eg. backend_timestamp). Will update the description to include this information throughout.